test/e2e/mxGraph.model.test.ts

```diff
@@ -32,6 +32,34 @@ export interface ExpectedFont {
   isStrikeThrough?: boolean;
 }
 
+export interface ExpectedShapeModelElement {
+  label?: string;
+  kind: ShapeBpmnElementKind;
+  font?: ExpectedFont;
+  /** Only needed when the BPMN shape doesn't exist yet (use an arbitrary shape until the final render is implemented) */
+  styleShape?: string;
+}
+
+export interface ExpectedEventModelElement extends ExpectedShapeModelElement {
+  eventKind: ShapeBpmnEventKind;
+}
+
+export interface ExpectedEventModelElement extends ExpectedShapeModelElement {
```

Comment:
Is it normal to have twice the same interface ?

Reply:
no :smiley_cat: 
I am on it

Reply:
done